src/app/armory/Links.tsx

```diff
@@ -0,0 +1,104 @@
+import { languageSelector } from 'app/dim-api/selectors';
+import ExternalLink from 'app/dim-ui/ExternalLink';
+import { DimItem } from 'app/inventory/item-types';
+import { LoreLink } from 'app/item-popup/ItemDescription';
+import { useIsPhonePortrait } from 'app/shell/selectors';
+import destinysets from 'images/destinysets.svg';
+import destinytracker from 'images/destinytracker.png';
+import logo from 'images/dimlogo.svg';
+import lightgg from 'images/lightgg.png';
+import React from 'react';
+import { useSelector } from 'react-redux';
+import styles from './Links.m.scss';
+
+// TODO: permalink for sharing
+const links = [
+  {
+    name: 'DIM',
+    icon: logo,
+    link: (item: DimItem) => `armory/${item.hash}`,
+  },
+  {
+    name: 'Light.gg',
+    icon: lightgg,
+    link: (item: DimItem, language: string) =>
+      `https://www.light.gg/db/${language}/items/${item.hash}`,
+  },
+  { name: 'DestinyTracker', icon: destinytracker, link: destinyDBLink },
+  {
+    name: 'data.destinysets.com',
+    icon: destinysets,
+    link: (item: DimItem, language: string) =>
+      `https://data.destinysets.com/i/InventoryItem:${item.hash}?lang=${language}`,
+    hideOnPhone: true,
+  },
+];
+
+export default function Links({ item }: { item: DimItem }) {
+  const language = useSelector(languageSelector);
+  const isPhonePortrait = useIsPhonePortrait();
+  return (
+    <ul className={styles.links}>
```

Comment:
the inconsistent underlined space here is a bit weird
![image](https://user-images.githubusercontent.com/68782081/133374150-2f495070-35d8-49d0-ab88-47b77a9f6cab.png)

Reply:
Yeah I wish links were a bit more subtle about underlining